### PR TITLE
feat: update fixtures to goerli fixtures

### PR DESCRIPTION
**Pull request by isaackps**
## Summary

Update fixtures as the fixtures are outdated.

## Changes

- addes goerli fixtures
- update test cases to use goerli fixtures

## Issues

https://www.pivotaltracker.com/n/projects/2424361/stories/183437253

### CODE REVIEW

File: src/common/hook/useQueue/utils/sample-config.json [hunk 1]
```diff
@@ -95,7 +95,7 @@
     }
   ],
   "documentStorage": {
-    "apiKey": "randonKey",
-    "url": "https://api-ropsten.tradetrust.io/storage"
+    "apiKey": "randomKey",
+    "url": "https://api-goerli.tradetrust.io/storage"
   }
```
Comment (simboonlong) [inline src/common/hook/useQueue/utils/sample-config.json:100]: maybe we should signal to users this is supposed to be their own setup. let's not put any "correct" looking url? ok to leave `url` to say for eg, `https://api.example.com/storage`?

File: src/types/index.d.ts [hunk 2]
```diff
@@ -1,9 +1,9 @@
-import { Wallet, Signer } from "ethers";
 import { Provider } from "@ethersproject/abstract-provider";
 import { OpenAttestationDocument } from "@govtechsg/open-attestation";
 import { ErrorObject } from "ajv";
+import { Signer, Wallet } from "ethers";
 
-export type Network = "homestead" | "ropsten" | "rinkeby" | "local";
+export type Network = "homestead" | "ropsten" | "rinkeby" | "goerli" | "local" | "sepolia" | "matic" | "maticmum";
 type FormType = "TRANSFERABLE_RECORD" | "VERIFIABLE_DOCUMENT";
```
Comment (simboonlong) [inline src/types/index.d.ts:7]: network variables are being repeated in `constants/chainInfo.tsx`, any way to consolidate them?

Comment (isaackps) [reply src/types/index.d.ts:7]: currently i cant leh, cos one is type and one is value.

**Review by simboonlong — APPROVED** [review]
i think ok, see if anyone else want to have 👀 too

**Pull request merged**